.eslintrc.json

```diff
@@ -10,7 +10,8 @@
 		"Cypress": "readonly",
 		"cy": "readonly",
 		"ajaxurl": "readonly",
-		"FormData": "readonly"
+		"FormData": "readonly",
+		"classifaiMediaVars": "readonly"
```

Comment:
`classifaiMediaVars` was removed in 1bfe51cc0a1b71187cefad44aefc28f18181a064

Reply:
@peterwilsoncc the PR wasn't up for re-review. I re-added `classifaiMediaVars` in  [ba3162a](https://github.com/10up/classifai/pull/374/commits/ba3162a273704b85f6efb4e105343d95572f6f56)